integration/simple-cache-client.test.ts

```diff
@@ -571,6 +573,223 @@ describe('lists', () => {
 });
 
 describe('Integration Tests for operations on sets datastructure', () => {
+  const LOL_BYTE_ARRAY = Uint8Array.of(108, 111, 108);
+  const FOO_BYTE_ARRAY = Uint8Array.of(102, 111, 111);
+  it('should succeed for addElements with byte arrays happy path', async () => {
+    const setName = v4();
+    const addResponse = await momento.setAddElements(
+      INTEGRATION_TEST_CACHE_NAME,
+      setName,
+      [LOL_BYTE_ARRAY, FOO_BYTE_ARRAY]
+    );
+    expect(addResponse).toBeInstanceOf(CacheSetAddElements.Success);
+
+    const fetchResponse = await momento.setFetch(
+      INTEGRATION_TEST_CACHE_NAME,
+      setName
+    );
+    expect(fetchResponse).toBeInstanceOf(CacheSetFetch.Hit);
+    expect((fetchResponse as CacheSetFetch.Hit).valueSetUint8Array()).toEqual(
+      new Set([LOL_BYTE_ARRAY, FOO_BYTE_ARRAY])
+    );
+  });
+  it('should succeed for addElements with byte arrays happy path with no refresh ttl', async () => {
+    const setName = v4();
+    let addResponse = await momento.setAddElements(
```

Comment:
I don't like that we are perpetuating this pattern of using mutable variables in all of these tests when they are not necessary, but not a blocker.